bar

```diff
@@ -0,0 +1 @@
+
```

Comment:
I'm not sure this is an adequate quantity of bar to make a sensible trade-off between foo and baz.